ArchiSteamFarm/Commands.cs

```diff
@@ -1169,6 +1173,47 @@ internal sealed class Commands {
 			return responses.Count > 0 ? string.Join(Environment.NewLine, responses) : null;
 		}
 
+		private async Task<string> ResponseLevel(ulong steamID) {
+			if (steamID == 0) {
+				Bot.ArchiLogger.LogNullError(nameof(steamID));
+				return null;
+			}
+
+			if (!Bot.IsConnectedAndLoggedOn) {
+				return FormatBotResponse(Strings.BotNotConnected);
+			}
+
+			if (!Bot.IsMaster(steamID)) {
+				return null;
+			}
+
+			uint? level = await Bot.ArchiHandler.GetLevel().ConfigureAwait(false);
+
+			if (level == null) {
```

Comment:
ASF has convention of using `HasValue` on nullable types, use it instead of `== null`.